packages/clerk-js/src/ui/components/OrganizationSwitcher/OtherOrganizationActions.tsx

```diff
@@ -95,6 +95,7 @@ export const OrganizationActionList = (props: OrganizationActionListProps) => {
             onClick={() => onOrganizationClick(organization)}
           >
             <OrganizationPreview
+              elementId={'organizationSwitcher'}
```

Comment:
Hey George, is this needed for now?

Reply:
Yes, it's something we forgot to add in a previous PR.